### DISCUSSION TITLE
TextArea in Table : Value not set???

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1592] [Q&A] [answered]
Hi

I have copied this fom the output of my html page.

https://play.tailwindcss.com/zs4HVXIV61

Fo some eason the text-aea value is not being set.

What can I be missing?

Thank you so much

**Comment by saadeghi** [comment, marked as answer]
HTML `<textarea>` has no `value` attribute. the value goes between `<textarea>` and `</textarea>`
```html
<textarea>VALUE</textarea>
```
